deploy/fb-user-datastore-chart/templates/deployment.yaml

```diff
@@ -23,6 +23,14 @@ spec:
         imagePullPolicy: Always
         ports:
           - containerPort: 3000
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 3000
+            scheme: HTTPS
```

Comment:
should this not be `http`? SSL is terminated outside of rails and i'm assuming the call is internal, therefore does not go through the ingress and will therefore be `http`?

Reply:
Good catch, cheers